core/modules/wiki.js

```diff
@@ -179,7 +179,7 @@ exports.generateNewTitle = function(baseTitle,options) {
 		title = baseTitle;
 	while(this.tiddlerExists(title) || this.isShadowTiddler(title) || this.findDraft(title)) {
 		title = baseTitle + 
-			(options.prefix || " ") + 
+			(options.prefix === undefined ? " " : options.prefix) + 
```

Comment:
@danielo515 no, `typeof(2) !== "string"`. So you'd have to pass a numeric prefix as a string, not a number (which is correct; the type is intended to be a string).